From d8191ec3679be777c0982c3a97602122e2ca2bee Mon Sep 17 00:00:00 2001
From: Tomi Ollila <tomi.ollila@iki.fi>
Date: Tue, 12 Nov 2024 21:54:21 +0200
Subject: [PATCH 2/7] declare `extern char **environ` in 'C' linkage

Same thing that was fixed in commit 78e41e4999da in another branch.
---
 src/main.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/main.h b/src/main.h
index e587478..33411e3 100644
--- a/src/main.h
+++ b/src/main.h
@@ -35,6 +35,10 @@
  * PUBLIC DECLARATIONS
  *============================================================================*/
 
+extern "C" {
+
 extern char** environ;
 
+}
+
 #endif
-- 
2.49.0

